.github/workflows/deploy.yml

```
@@ -0,0 +1,62 @@
+
+    - name: Deploy Bicep template
+      run: |
+        az deployment sub create \
+          --name "swa-deployment-$(date +%s)" \
+          --location westus2 \
+          --template-file infra/main.bicep \
+          --parameters resourceGroupName=${{ secrets.RESOURCE_GROUP_NAME }} location=westus2
+
+    - name: Get Static Web App deployment token
+      id: get-token
+      run: |
```

actionlint (1.7.7)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)